internal/page/data.go

```diff
@@ -70,14 +71,16 @@ const (
 // Lpa contains all the data related to the LPA application
 type Lpa struct {
 	PK, SK string
+	// Hash is used to determine whether the Lpa has been changed since last read
+	Hash uint64 `hash:"-"`
```

Comment:
I'm trying to not ignore too many fields, only those that contain hash values or are updated in `donorStore.Put` after the hash field is set